README.md

```diff
@@ -79,10 +84,110 @@ Returns a comma-separated list of all values for this header name or an empty st
 The `hasHeader(string $name): bool` method can be used to
 check if a header exists by the given case-insensitive name.
 
+#### expectsContinue()
+
+The `expectsContinue(): bool` method can be used to
+check if the request headers contain the `Expect: 100-continue` header.
+
+This header MAY be included when an HTTP/1.1 client wants to send a bigger
+request body.
+See [`writeContinue()`](#writecontinue) for more details.
+
 ### Response
 
+The `Response` class is responsible for streaming the outgoing response body.
+
+It implements the `WritableStreamInterface`.
+
 See the above usage example and the class outline for details.
 
+#### writeContinue()
+
+The `writeContinue(): void` method can be used to
+send an intermediary `HTTP/1.1 100 continue` response.
+
+This is a feature that is implemented by *many* HTTP/1.1 clients.
+When clients want to send a bigger request body, they MAY send only the request
+headers with an additional `Expect: 100-continue` header and wait before
+sending the actual (large) message body.
+
+The server side MAY use this header to verify if the request message is
+acceptable by checking the request headers (such as `Content-Length` or HTTP
+authentication) and then ask the client to continue with sending the message body.
+Otherwise, the server can send a normal HTTP response message and save the
+client from transfering the whole body at all.
+
+This method is mostly useful in combination with the
+[`expectsContinue()`](#expectscontinue) method like this:
+
+```php
+$http->on('request', function (Request $request, Response $response) {
+    if ($request->expectsContinue()) {
+        $response->writeContinue();
+    }
```

Comment:
If I write code like documented now, how does reading the request body work then? Do I have to find it out manually and discard the first expect 100 request?